tutorials/model_command.md

```diff
@@ -36,29 +27,31 @@ This command allows you to get the following information about models in the run
 
 ## Example running the diff_drive world
 
-After having an Ignition Gazebo server running with the diff_drive world, you may check the available models running `ign model --list`
+To try out this command we need first a running simulation. Let's load the diff_drive ignition simulation. In a terminal, run:
```

Comment:
nit: use **diff_drive** instead of plain diff_drive

Reply:
#928